source/read/count.txt

```diff
@@ -0,0 +1,230 @@
+.. _cpp-count:
+
+===============
+Count Documents
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: number, amount, estimation, code example
+
+Overview
+---------
+
+In this guide, you can learn how to use the {+driver-short+} to retrieve an accurate
+and estimated count of the number of documents in a collection. The ``count_documents()``
+method returns the exact number of documents that match a query filter or that exist in
+a collection, and the ``estimated_document_count()`` method returns the estimated number
+of documents in a collection.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``companies`` collection in the ``sample_training``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
+from your C++ application, instantiate a ``mongocxx::client`` that connects to an Atlas
+cluster and assign the following values to your ``db`` and ``collection`` variables:
+
+.. literalinclude:: /includes/read/count.cpp
+    :language: cpp
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<cpp-get-started>` tutorial.
```

Comment:
Is this reference correct? I do not think the "Get Started with the C++ Driver" pages are relevant to creating a free MongoDB Atlas cluster.

Reply:
The Get Started tutorial includes a page ([Create a MongoDB Deployment](https://preview-mongodbnorareidy.gatsbyjs.io/cpp-driver/DOCSP-40801-count/get-started/create-a-deployment/)) that walks you through how to create an Atlas cluster. However it does link out to the Atlas docs for details, so I can link directly to the Atlas page here instead if you'd prefer

Reply:
Ok! I think it makes sense to link to the Atlas page; updated